cvmfs/cvmfs.cc

```diff
@@ -1814,8 +1814,18 @@ static int Init(const loader::LoaderExports *loader_exports) {
   struct fuse_chan **channel = NULL;
   if (loader_exports->version >= 4)
     channel = loader_exports->fuse_channel;
-  cvmfs::fuse_remounter_ = new FuseRemounter(
-    cvmfs::mount_point_, &cvmfs::inode_generation_info_, channel);
+
+  bool use_fuse_notify_invalidation = true;
+  std::string buf;
+  if (cvmfs::options_mgr_->GetValue("CVMFS_FUSE_NOTIFY_INVALIDATION",
+                                    &buf)) {
+    if (buf == "false") {
```

Comment:
We could check for  `!cvmfs::options_mgr_::IsOn(...)`, which understands all of `true`, `1`, `on`, etc.